static/js/containers/DashboardPage.js

```diff
@@ -126,6 +135,26 @@ class DashboardPage extends React.Component {
     }
   };
 
+  updateRequirements = () => {
+    this.fetchDashboard();
+    this.fetchCoursePrices();
+    this.handleOrderStatus();
+  };
+
+  fetchDashboard() {
+    const { dashboard, dispatch } = this.props;
+    if (dashboard.fetchStatus === undefined) {
```

Comment:
It's not possible for globals to be replaced in that matter, only locals: http://www.2ality.com/2013/04/check-undefined.html